crates/driver/src/api/solve/dto/auction.rs

```diff
@@ -1,84 +1,211 @@
+                        partial: if order.partially_fillable {
+                            competition::order::Partial::Yes {
+                                executed: order.executed.into(),
+                            }
+                        } else {
+                            competition::order::Partial::No
+                        },
+                        interactions: order
+                            .interactions
+                            .into_iter()
+                            .map(|interaction| eth::Interaction {
+                                target: interaction.target.into(),
+                                value: interaction.value.into(),
+                                call_data: interaction.call_data,
+                            })
+                            .collect(),
+                        sell_token_balance: match order.sell_token_balance {
+                            SellTokenBalance::Erc20 => competition::order::SellTokenBalance::Erc20,
+                            SellTokenBalance::Internal => {
+                                competition::order::SellTokenBalance::Internal
+                            }
+                            SellTokenBalance::External => {
+                                competition::order::SellTokenBalance::External
+                            }
+                        },
+                        buy_token_balance: match order.buy_token_balance {
+                            BuyTokenBalance::Erc20 => competition::order::BuyTokenBalance::Erc20,
+                            BuyTokenBalance::Internal => {
+                                competition::order::BuyTokenBalance::Internal
+                            }
+                        },
+                        signature: competition::order::Signature {
+                            scheme: match order.signing_scheme {
+                                SigningScheme::Eip712 => {
+                                    competition::order::signature::Scheme::Eip712
+                                }
+                                SigningScheme::EthSign => {
+                                    competition::order::signature::Scheme::EthSign
+                                }
+                                SigningScheme::PreSign => {
+                                    competition::order::signature::Scheme::PreSign
+                                }
+                                SigningScheme::Eip1271 => {
+                                    competition::order::signature::Scheme::Eip1271
+                                }
+                            },
+                            data: order.signature,
+                            signer: order.owner.into(),
+                        },
+                        reward: order.reward,
+                    })
+                })
+                .try_collect()?,
+            // TODO #899
+            liquidity: Default::default(),
```

Comment:
I don't think that the protocol will ever send liquidity as part of the `Auction` DTO - I only think this exists in the driver <> solver `Auction` DTO (as it is populated by the driver, and doesn't come from the protocol). I think we can remove it here.

Reply:
I see it was add in #999 in the solver DTO (where I think it should be) - so I would remove this from here.

Reply:
I understood that this was the DTO that was being sent from the `autopilot` to the `driver` which doesn't have liquidity 🤦.